docs/guides/cli/17_configure-permission-service.mdx

```diff
@@ -77,3 +79,61 @@ ory patch permission-config <project-id> \
 Read [this document](./config-with-cli) to learn more about adjusting project configuration with the Ory CLI.
 
 :::
+
+## Manage relation tuples
+
+You can manage the [relation tuples](../../keto/concepts/relation-tuples) directly from the Ory CLI as well. Here's how:
+
+### Setup
+
+```bash
+export PROJECT=<your_project_id>
+
+## Authenticate first
+ory auth
+# (sign in with Ory Cloud account)
+```
+
```

Comment:
This paragraph seems to be concerned with setting up Ory CLI. Can you move it out of the "Manager relation tuples" chapter into a "setup Ory CLI" chapter and link to this chapter from here?